.platform/hooks/prebuild/08_link_build_caches.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
-cd "$(/opt/elasticbeanstalk/bin/get-config platformconfig -k AppStagingDir)"
+if ! [ -d /var/app/bundle ]; then
```

Comment:
Is it going to be safe to share all these artefacts across instances? For example aren't there native extensions built into this? Just wondering if there could be an issue across versions of Amazon Linux